packages/hash/design-system/src/ui.ts

```diff
@@ -0,0 +1,19 @@
+import "./theme-override.d";
+
+export * from "./ui/avatar";
+export * from "./ui/button";
+export * from "./ui/icon-button";
+export * from "./ui/loading-spinner";
+export * from "./ui/popover";
+export * from "./ui/submit-button-wrapper";
+export * from "./ui/text-field";
+export * from "./ui/form-inline";
+export * from "./ui/chip";
+export * from "./ui/chip-group";
+export * from "./ui/menu-item";
+export * from "./ui/menu-checkbox-item";
+export * from "./ui/select";
+export * from "./ui/fontawesome-icon";
+
+export * from "./ui/theme";
+export * from "./ui/utils";
```

Comment:
Seems like pretty much all we have in `src` is `ui` folder. How about hoisting its contents?

Reply:
yep, thanks!